avalanche-network-runner-sdk/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "avalanche-network-runner-sdk"
 version = "0.3.2" # https://crates.io/crates/avalanche-network-runner-sdk
```

Comment:
your going to need to push a new crate version for this to work right?

Reply:
Yes, that's a good point. Since I'm not a maintainer I will wait until @gyuho can take a look.